src/scripts/create_cnn_dailymail.py

```diff
@@ -0,0 +1,65 @@
+"""Create the CNN-DailyMail-mini summarisation dataset."""
```

Comment:
For Scandinavian datasets there haven't been much of a discussion about which datasets to use and why, but it might be ideal to discuss it here where we have multiple datasets to choose from. We even have the option to balance the dataset across a wide variety of domains, which might give us a more representative estimate. 

CNN is fairly similar to the other summarization datasets (newslike), but are we actually interested in news summarization or summarization in general (I would say in general).

That being said It might be worth simply creating an issue and merging this PR as is.

Reply:
E.g. for DaNE it might also be an idea to shuffle in parts of DANSK.

Reply:
Yeah you're right that the choice is currently somewhat arbitrary (mostly chosen based on similarity of domains and format). 

They can definitely be changed. I would suggest that such a change will be done in one PR, which would result in a major change to the package. Also, the leaderboard would need to be redone after such a change (probably 1-3 months of GPU hours needed for this).

Reply:
> Yeah you're right that the choice is currently somewhat arbitrary (mostly chosen based on similarity of domains and format).

Exactly, but I think it would require more discussion than this PR allows. There is a tradeoff between making the two languages comparable (Should NE for Danish should be similar to NE for Norwegian or is the goal making the best estimator for NE for Danish).

> They can definitely be changed. I would suggest that such a change will be done in one PR, which would result in a major change to the package. Also, the leaderboard would need to be redone after such a change (probably 1-3 months of GPU hours needed for this).

Wouldn't you rather do it a dataset at a time? Incremental changes are easier to manage and it doesn't really change the compute time (you only need to rerun that specific dataset to update the benchmark). Of course, the man-hours might be a reason to do it differently.

Reply:
You're right that doing it one dataset at a time is easiest if compute is flexible. But if we have to benchmark big generative models with, e.g., Peter's compute, then I'm guessing it would be easier to do it all at once. But maybe I'm wrong here!

As for the goal of the benchmark, I'll create a separate discussion here on this repo for that 🙂

Reply:
@peter-sk ?

Reply:
Created a discussion here:

https://github.com/ScandEval/ScandEval/discussions/85

I'll create separate discussions for the development of the new versions of (some of the) datasets.

Reply:
Here's a discussion regarding a new Danish NER dataset:
https://github.com/ScandEval/ScandEval/discussions/86